client/plots/matrix.cells.js

```diff
@@ -39,9 +39,13 @@ function setNumericCellProps(cell, tw, anno, value, s, t, self, width, height, d
 			cell.height = t.scale(cell.key)
 			cell.x = tw.settings.gap // - cell.width
 		} else {
-			cell.height = cell.key >= 0 ? t.scales.pos(cell.key) : t.scales.neg(cell.key)
+			const vc = cell.term.valueConversion
+			const renderV = vc ? cell.key * vc.scaleFactor : cell.key
+			cell.height = cell.key >= 0 ? t.scales.pos(renderV) : t.scales.neg(renderV)
 			cell.x = cell.totalIndex * dx + cell.grpIndex * s.colgspace
-			cell.y = cell.key >= 0 ? t.counts.posMaxHt + t.tw.settings.gap - cell.height : 0
+			cell.y =
+				cell.key >= 0 ? t.counts.posMaxHt + t.tw.settings.gap - cell.height : t.counts.posMaxHt + t.tw.settings.gap
+			cell.convertedValueLabel = vc && t.scales.format ? t.scales.format(renderV) + ' ' + vc.unit : ''
```

Comment:
i realized this method does not yield a label of `X years, Y days`. can that be done?